sql-plugin/src/main/java/com/nvidia/spark/rapids/InternalRowToColumnarBatchIterator.java

```diff
@@ -147,55 +147,61 @@ public ColumnarBatch next() {
     // Update our estimate for number of rows with the final size used to allocate the buffers.
     numRowsEstimate = (int) bufsAndNumRows._2.targetSize();
     long dataLength = calcDataLengthEstimate(numRowsEstimate);
-    try (
-        SpillableHostBuffer sdb = bufsAndNumRows._1[0];
+    int used[];
+    try (SpillableHostBuffer sdb = bufsAndNumRows._1[0];
```

Comment:
Not introduced by this PR but since it touches them, can we update these variables to be more mnemonic than `sdb` and `sob` to improve code readability